docs/API.md

```diff
@@ -567,8 +567,8 @@ An object with the property:
 
 An object with the following properties:
 
-* `"ns_list"`: A list of *name server* objects representing the nameservers of the given *domain name*.
-* `"ds_list"`: A list of *DS info* objects representing delegated signer of the given *domain name*.
+* `"ns_list"`: A list of [*name server*][Name server] objects representing the nameservers of the given *domain name*.
+* `"ds_list"`: A list of [*DS info*][DS info] objects representing delegated signer of the given *domain name*.
```

Comment:
...delegation signer (DS record data) of the given...

Reply:
updated.